### ISSUE TITLE
Validate all images are actually images during the tests

### ISSUE THREAD

**Issue by MarshallOfSound** [electron/electron-apps#431]
We could do a `HEAD` request to all image URL's to ensure the mimeType returned is actually an image.

This would prevent things like #430 where the URL added didn't actually point to an image 👍 

Existing tests: https://github.com/electron/electron-apps/blob/master/test/human-data.js#L100

**Comment by zeke** [comment]
Yeah I had a similar thought a while back: https://github.com/electron/electron-apps/issues/88

We'd need to make sure this check only happens on newly-added (or newly updated) apps, so as to avoid making a jillion network requests. :)

**Comment by as900900** [comment]
مشكرين. .
